packages/mergebot/src/scripts/updateJSONFixtures.mjs

```diff
@@ -0,0 +1,50 @@
+// @ts-check
```

Comment:
This is now mjs; was this script never referenecd elsewhere? Or is this a new thing entirely form the old one? Is there a doc that needs to be updated?

Reply:
Oh, the old one was some travis thing :smile:

Reply:
It used to be updateJSONFixtures.ts, and run with ts-node. It's not referenced anywhere else, and not documented.

Reply:
There's no reason for it to be .ts except to use module syntax, and it's way easier to run (with node >=20) as a JS file.